### ISSUE TITLE
Bot fails to get information for several wikis, giving weird errors and no failing links

### ISSUE THREAD

**Issue by ciencia** [WikiApiary/WikiApiary#218]
I've seem several errors in logs from WMF wikis. They contain weird errors:
- Unhandled exception must be string, not None during unknown [ link]
- unknown url type: ?action=query&meta=siteinfo&siprop=statistics&format=json [?action=query&meta=siteinfo&siprop=statistics&format=json link]
- Unhandled exception unsupported operand type(s) for +: 'NoneType' and 'str'. [ link]

Examples:

https://wikiapiary.com/wiki/Wikipedia_%28to%29
https://wikiapiary.com/wiki/Wikisource_%28zh-min-nan%29

**Comment by kghbln** [comment]
Yeah, this was discussed on the wikiapiary mailing list a week or so ago at least for point two and three. The issue is that for these wikis the API URL annotations were lost somehow. After updating both MW and SMW  [trivial edits to these wiki's pages](https://wikiapiary.com/w/index.php?title=Wikisource_%28zh-min-nan%29&diff=1596318&oldid=706504) should resolve the issue.
